share/bash/control-pelagos-consumers.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+#
+# chkconfig: 35 98 02
+# description: Pelagos RabbitMQ Consumer Supervisor
+#
+# This script to be copied to or symlinked at /usr/local/bin and set to executable.
+#
+# This script is called by the pelagos-consumer.service service with either a
+# start or stop argument.
+
+# Get function from functions library
+. /etc/init.d/functions
+
+prog="pelagos-rabbitmq-consumer-supervisor"
+basecmd="bin/console rabbitmq-supervisor:control"
+runuser=pelagos
+postcmd="--wait-for-supervisord"
+
+# Start the service
+start() {
+        echo -n $"Starting $prog: "
+	cd /opt/pelagos
+        if su - $runuser -c "$basecmd start $postcmd" ; then
+            success
+        else
+            failure
+        fi
+        echo
+}
+
+# Stop the service
+stop() {
+	cd /opt/pelagos
+	pwd
+        echo -n $"Stopping $prog: "
+        if su - $runuser -c "$basecmd stop $postcmd" ; then
```

Comment:
I don't think we need the postcmd for stopping the service. It works without it as well

Reply:
doesn't seem to always though

Reply:
I tested stopping without the wait for supervisord, it works but it doesn't hurt to have that